IOSExercises/main.m

```diff
@@ -0,0 +1,28 @@
+//
+//  main.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 10/28/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "AppDelegate.h"
+#import "Classes/LocaleManager.h"
+
+int main(int argc, char * argv[]) {
+    @autoreleasepool {
+        [[NSUserDefaults standardUserDefaults] removeObjectForKey:@"AppleLanguages"];
+        NSString *appLanguage = [[LocaleManager getManager] getPreferredLanguage];
+
+        if (appLanguage == nil) {
+            NSLog(@"Region is not supported. Setting English as default language.");
```

Comment:
NSLog(@"main: Region is not supported. Setting English as default language.");

Start with class name